addons/l10n_cn/static/src/js/action_manager_report.js

```diff
@@ -0,0 +1,29 @@
+odoo.define('l10n_cn.ReportActionManager', function (require) {
+"use strict";
+/**
+ * The purpose of this file is to check if cn2an is installed.
+ */
+
+const ActionManager = require('web.ActionManager');
+const core = require('web.core');
+
+const _t = core._t;
+
+ActionManager.include({
+    _executeReportAction: function (action, options) {
+        if (action.xml_id && action.xml_id == "l10n_cn.account_voucher_cn") {
+            this._rpc({
+                method: 'check_cn2an',
+                model: 'account.move',
+            }).then(res => {
+                if (!res) {
+                    var msg = _t("Unable to find cn2an on this system. The \"amount in words\" won't be shown.") + '<br><br><a href="https://pypi.org/project/cn2an/" target="_blank">cn2an · PyPI</a>';
+                    this.do_notify(_t('Voucher'), msg);
+                }
```

Comment:
You could just have put nothing in the QWeb report and we describe this in the manifest somehow.  It is nice, but a bit overkill.  If we install it on the SaaS (I also think this is backportable to v14) they won't need to install manually anyways. @yhu-odoo

Reply:
Sorry I forget to remove this part. You could just remove this file and adapt the manifest. I just checked, it works fine.